app/jobs/create_uphold_cards_job.rb

```diff
@@ -2,24 +2,23 @@
 class CreateUpholdCardsJob < ApplicationJob
   queue_as :default
 
-  def perform(publisher_id:)
-    publisher = Publisher.find(publisher_id)
-
-    unless publisher.uphold_connection.can_create_uphold_cards?
-      Rails.logger.info("Could not create uphold card for publisher #{publisher.id}.")
-      SlackMessenger.new(message: "Could not create uphold card for publisher #{publisher.id}.", channel: SlackMessenger::ALERTS).perform
+  def perform(uphold_connection:)
```

Comment:
You need to lookup the `uphold_connection` by id since Sidekiq jobs only allow primitives (and dynos don't know each other memory space)

Reply:
https://github.com/brave-intl/publishers/pull/2062/commits/4929743cb20a2bffbaed09e9628689bf04e53d1d